styles.css

```diff
@@ -0,0 +1,85 @@
+@import url('https://fonts.googleapis.com/css2?family=Alumni+Sans+Pinstripe&family=Poppins&display=swap');
+
+#toolbar {
+  display: flex;
+  position: fixed;
+  height: 35px;
+  width: 920px;
+  padding: 10px;
+}
+
+#logo {
+  color: #6070ff;
+  margin-right: 125px;
+  font-size: large;
+  font-weight: bold;
+}
+
+.nav-item {
+  display: flex;
+  margin-left: 10px;
+  font-family: 'Poppins', sans-serif;
+}
+
+#headline {
+  display: flex;
+  flex-direction: column;
+  height: 740px;
+  width: 320px;
+  background-image: url("assets/images/mobile.png");
```

Comment:
- [ ] [ OPTIONAL ] Consider removing a fixed width from your declaration as this is not the best practice in responsive web design.  In the image below I crossed out the width. Try it and see the difference. 
    ![image](https://user-images.githubusercontent.com/84156714/185713841-6dc1cff2-544d-4e9b-92f0-6673801ecfbc.png)

Reply:
I appreciate this suggestion but since it is optional I will skip it now because I am running out of time. But in the future I will consider the suggestion.